### DISCUSSION TITLE
Private Message Formatting

### DISCUSSION THREAD

**Discussion by Coadon** [EssentialsX/Essentials#4338] [Ideas and Feature Suggestions] [answered]
Would it be nice if we can format private messages, such as /msg or /tell? Since we can already format how "public" chats should be represented (which is nice too,) why not make that happen as well?
It will be even nicer if we can customize how the recipient should receive the message specifically, and the also sender specifically.

**Comment by Coadon** [comment]
Sorry, I got a grammar issue there, it should be "and also the sender specifically."

**Comment by pop4959** [comment, marked as answer]
You can do this already using a custom message file. See [here](https://essentialsx.net/wiki/Locale.html#customising-messages) for more info on setting that up.

**Comment by Coadon** [comment]
Ok, thanks, it worked, and I guess I have done something in a way that is so much harder, while I can literally use that method instead. :\ good to know.
